core/test/unit/middleware/spam-prevention_spec.js

```diff
@@ -19,6 +20,7 @@ describe('Middleware: spamPrevention', function () {
         spyNext = sinon.spy(function (param) {
```

Comment:
Sorry, I don't know much on how to improve this test, just enough to get it pass on my machine. May be you should take from there, a brand new PR whould be welcome.